### DISCUSSION TITLE
UNO - Windows app does not detect [NotifyPropertyChangedFor())

### DISCUSSION THREAD

**Discussion by mcNets** [CommunityToolkit/dotnet#466] [Q&A]
Is this example border background should change when IsPositive is true.

It works fine on Skia.Gtk, but it doesn't work on Windows.

I don't know if it is related to this: https://github.com/unoplatform/uno/discussions/9876

I've added a minimal example. (Uno App -> Windows + Skia)

XAML

```
<Border x:Name="ST_Border"
        Background="Transparent"
        Height="100">

    <VisualStateManager.VisualStateGroups>
        <VisualStateGroup x:Name="Stopped">
            <VisualState x:Name="On">
                <VisualState.StateTriggers>
                    <StateTrigger IsActive="{x:Bind ViewModel.IsPositive, Mode=OneWay }" />
                </VisualState.StateTriggers>
                <VisualState.Setters>
                    <Setter Target="ST_Border.Background"
                            Value="Blue" />
                </VisualState.Setters>
            </VisualState>
        </VisualStateGroup>
    </VisualStateManager.VisualStateGroups>

....


```

ViewModel:

```
public partial class MainPageViewModel : ObservableObject
{
    [ObservableProperty]
    [NotifyPropertyChangedFor(nameof(IsPositive))]
    private int _stopTime = 0;

    public bool IsPositive => StopTime > 0;

    public void Increase()
    {
        StopTime += 10;
    }

    public void Decrease()
    {
        StopTime -= 10;
    }
}
```
[TestCommunityToolkit.zip](https://github.com/CommunityToolkit/WindowsCommunityToolkit/files/9676107/TestCommunityToolkit.zip)

**Comment by michael-hawker** [comment]
@mcNets might be best to ask this in the [Uno discussions](https://github.com/unoplatform/uno/discussions)?

**Reply by mcNets** [reply]
Hi @michael-hawker , I did and they suggested to open a discussion in WinUI.

https://github.com/unoplatform/uno/discussions/10018#discussioncomment-3763734

**Reply by michael-hawker** [reply]
Moved this over to the .NET Community Toolkit discussions where the MVVM Toolkit resides. 

Pulled the zip up and saw this is Windows App SDK 1.1.3. FYI @Sergio0694 Jerome had some thoughts in the uno copy of this discussion. I don't think we've noticed any issues though. Thoughts?
